src/js/grapl-cdk/lib/dgraph.ts

```diff
@@ -115,38 +115,43 @@ class Alpha extends cdk.Construct {
     }
 }
 
+export interface DGraphEcsProps {
```

Comment:
This will need to be updated to contain `alphaPort` after merging staging into this branch.

Reply:
👍 merge conflict forced my hand